src/get-started/create-service.md

```diff
@@ -748,22 +748,49 @@ let peer_address = "0.0.0.0:2000";
 // Returns the value of the `NodeConfig` object from the `node_config` function
 NodeConfig {
     listen_address: peer_address.parse().unwrap(),
-    service_public_key,
-    service_secret_key,
-    consensus_public_key,
-    consensus_secret_key,
-    genesis,
+    consensus,
     external_address: peer_address.to_owned(),
     network: Default::default(),
     connect_list: Default::default(),
     api: api_cfg,
     mempool: Default::default(),
-    services_configs: Default::default(),
-    database: Default::default(),
     thread_pool_size: Default::default(),
+    keys: Keys::from_keys(
+        consensus_public_key,
+        consensus_secret_key,
+        service_public_key,
+        service_secret_key,
+    ),
 }
 ```
 
+## Create Genesis Configuration
+
+`NodeConfig` that we created earlier defines the configuration for the **node**,
+but we also need a configuration for the **blockchain**. To initialize
+the blockchain, we need `GenesisConfig` structure.
+
+Let's create a function which will generate `GenesisConfig` based on
+the created `NodeConfig`:
+
+```rust
+fn genesis_config(node_config: &NodeConfig) -> GenesisConfig {
+    let artifact_id = CryptocurrencyService.artifact_id();
+    GenesisConfigBuilder::with_consensus_config(node_config.consensus.clone())
+        .with_artifact(ExplorerFactory.artifact_id())
+        .with_instance(ExplorerFactory.default_instance())
+        .with_artifact(artifact_id.clone())
+        .with_instance(artifact_id.into_default_instance(101, "cryptocurrency"))
+        .build()
+}
+```
+
+In the code above we create `GenesisConfig` from the consensus configuration,
+and add the `Explorer` and our `Cryptocurrency` services.
+
+`Explorer` service is capable of sending transactions to the blockchain, so
```

Comment:
Nit: It could make sense to link to the explorer service description in *Other Services* article.